lib/v2/gocn/jobs.js

```diff
@@ -1,23 +0,0 @@
-const got = require('@/utils/got');
```

Comment:
Consider replacing it with content from `https://gocn.vip/c/3lQ6GbD5ny/s/Gd7OHl` instead of removing it.

Reply:
I will try to fix it.